src/components/StudyRoom/StudyRoomCreate.js

```diff
@@ -0,0 +1,195 @@
+import {PrimaryButton} from "../CustomMUIComponents/CustomButtons";
+import MenuItem from "@mui/material/MenuItem";
+import CircleIcon from '@mui/icons-material/Circle';
+import "./customButton.css"
+import axios from "axios";
+import {useNavigate} from "react-router";
+import {useState} from "react";
+import mockUser from "./../Profile/Mocks/mockUser.json"
+import {ParticipantCard} from "./StudyRoomCards";
+
+//friend list from the user mock data
+const loggedInUser = mockUser[1].friends;
+console.log(loggedInUser);
+
+export default function RoomCreation() {
+    const navigate = useNavigate();
+    const [errorMessage, setErrorMessage] = useState('')
+    const [checked, setChecked] = React.useState([]);
+    const [roomData, setRoomData] = React.useState({
+        title:'',
+        color: '',
+        avatarText:'',
+        description:'',
+        participants:[],
+    });
+
+    React.useEffect(()=> {
+        let user = localStorage.getItem("username")
+        setRoomData({...roomData, owner: user})
+        })
+
+    React.useEffect( ()=>{
+        fetchData();
+    },[])
+    function fetchData() {
+        axios.get('http://localhost:5000/friend/murth')
+            .then(res => {
+                console.log(res);
+                console.log(res.body)
+            })
+    }
+
+    const handleCheck =(e) =>{
+        let updatedList = [...checked];
+        if(e.target.checked){
+            updatedList = [...checked, e.target.value];
+        }else{
+            updatedList.splice(checked.indexOf(e.target.value), 1);
+        }
+        setChecked(updatedList);
+        setRoomData({...roomData, participants:updatedList});
+    }
+
+    const handleRoomCreation = (e) =>{
+        e.preventDefault();
+        let avatarIconText = SetAvatarText(roomData.title);
+        setRoomData({...roomData, avatarText: avatarIconText});
+
+        axios.post('http://localhost:5000/room/',roomData)
+            .then(res => {
+                console.log(res);
+                navigate("/study-room-home");
+            })
+            .catch(err => {console.log(`Error: ${err}`); setErrorMessage(`${err}`.substring(44) === 401 ? 'request could not be sent' : `${err}`)});
+    }
+
+    function handleTitleChange(e){
+        let avatar = SetAvatarText( e.target.value)
+        setRoomData({...roomData, title: e.target.value, avatarText:avatar});
+    }
+    function handleColorChange(e){
+        setRoomData({...roomData, color: e.target.value});
+    }
+    function handleDescriptionChange(e){
+        setRoomData({...roomData, description: e.target.value});
+    }
+
+    function SetAvatarText(t){
+        let finalText='';
+        let displayText = t.split(' ').map(word => word[0]).join('');
+
+        if(displayText.length >4){
+            finalText = displayText.substring(0,4);
+        }else
+            finalText = displayText;
+        return finalText;
+    }
+
+    const createRoom = (
+        <React.Fragment>
```

Comment:
Can you wrap this in a div with responsive height (instead of the React.Fragment)? The issues depicted in the photos seem to be related to responsiveness.